firmware/src/networking/wifi/esp.rs

```diff
@@ -39,11 +39,12 @@ pub async fn network_task() {
 
 	let mut buffer = [0u8; 256];
 	let mut i = 0;
+	socket.bind(25565).unwrap();
```

Comment:
haHAA funny minecraft number 😂😂😂😂😂😂😂🍆😳

Reply:
this was actually a bug, because the socket wasnt binded. We couldnt actually talk with others XD